mungegithub/mungers/close-stale.go

```diff
@@ -109,6 +99,20 @@ func findLastHumanPullRequestUpdate(obj *github.MungeObject) (*time.Time, bool)
 	return lastHuman, true
 }
 
+func findLastHumanPullRequestUpdate(obj *github.MungeObject) (*time.Time, bool) {
+	pr, ok := obj.GetPR()
+	if !ok {
+		return nil, ok
+	}
+
+	comments, ok := obj.ListReviewComments()
+	if !ok {
+		return nil, ok
+	}
+
+	return findLastHumanPullRequestUpdateHelper(pr.CreatedAt, comments)
```

Comment:
@apelisse was suggesting that the entire Helper function and call can be replaced with
```
import c "k8s.io/test-infra/mungegithub/mungers/matchers/comment"
...
c.LastComment(comments, c.HumanActor(), prCreatedAt)
```